README.md

```diff
@@ -0,0 +1,30 @@
+# terraform-aws-ecr-repository
+A Terraform module to create an Amazon Web Services (AWS) Elastic Container Registry (ECR) repository.
+
+## Usage
+
+```hcl
+module "ecr-repository" {
+  source  = "azavea/ecr-repository/aws"
```

Comment:
I think that we should keep the example `source` reference consistent with the existing modules:

```hcl
module "ecr-repository" {
  source = "github.com/azavea/terraform-aws-ecr-repository?ref=0.1.0"
  ...
```

Reply:
Ok. That was generated by the Terraform Registry. I like the `GitHub.com` notation better.